pkg/activator/activator.go

```diff
@@ -168,7 +164,9 @@ func (a *activator) addAPI(obj interface{}) {
 	a.logger.Debugw("adding new api activator", zap.String("apiName", apiName))
```

Comment:
Should we use the `a.activatorsMux` lock for this entire function?

Reply:
not necessary, it is better as it is, in terms of performance